pkg/datapath/linux/config/config.go

```diff
@@ -205,6 +205,7 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	cDefinesMap["LB4_BACKEND_MAP"] = "cilium_lb4_backends_v3"
 	cDefinesMap["LB4_REVERSE_NAT_SK_MAP"] = lbmap.SockRevNat4MapName
 	cDefinesMap["LB4_REVERSE_NAT_SK_MAP_SIZE"] = fmt.Sprintf("%d", lbmap.MaxSockRevNat4MapEntries)
+	cDefinesMap["IS_L3_DEV(ifindex)"] = "false"
```

Comment:
Why is this needed? That macro is already defined lower in the same function.

Reply:
Removed, thanks for reminding.